components/compliance-service/ingest/ingestic/ingestic.go

```diff
@@ -345,6 +345,14 @@ func (backend *ESClient) setYesterdayLatestToFalse(ctx context.Context, nodeId s
 	oneDayAgo := time.Now().Add(-24 * time.Hour)
 	indexOneDayAgo := mapping.IndexTimeseriesFmt(oneDayAgo)
 
+	today := time.Now()
+	indexToday := mapping.IndexTimeseriesFmt(today)
+
+	indexes, err := backend.client.IndexNames()
```

Comment:
here you are fetching all index names, and trying to modify the data on those indexes. That will be a performance hit. So try to modify the data for the last 90 days.